container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/engine_functions/create_engine.go

```diff
@@ -111,20 +107,6 @@ func CreateEngine(
 			removeLogsAggregatorFunc()
 		}
 	}()
-
-	// schedule log removal for log retention
-	go func() {
```

Comment:
is this code remotion fine? just to verify that it's not an accidental remotion because I didn't see something in the description

Reply:
yes it is, I moved it into the `engine` (which is actually where it should be)